firewalld/config.json.template

```diff
@@ -0,0 +1,388 @@
+					"CAP_LEASE",
+					"CAP_AUDIT_WRITE",
+					"CAP_AUDIT_CONTROL",
+					"CAP_SETFCAP",
+					"CAP_DAC_OVERRIDE",
+					"CAP_MAC_OVERRIDE",
+					"CAP_DAC_READ_SEARCH",
+					"CAP_MAC_ADMIN",
+					"CAP_SYSLOG",
+					"CAP_WAKE_ALARM",
+					"CAP_BLOCK_SUSPEND",
+					"CAP_AUDIT_READ"
+				],
+				"ambient": [
+					"CAP_CHOWN",
+					"CAP_FOWNER",
+					"CAP_FSETID",
+					"CAP_KILL",
+					"CAP_SETGID",
+					"CAP_SETUID",
+					"CAP_SETPCAP",
+					"CAP_LINUX_IMMUTABLE",
+					"CAP_NET_BIND_SERVICE",
+					"CAP_NET_BROADCAST",
+					"CAP_NET_ADMIN",
+					"CAP_NET_RAW",
+					"CAP_IPC_LOCK",
+					"CAP_IPC_OWNER",
+					"CAP_SYS_MODULE",
+					"CAP_SYS_RAWIO",
+					"CAP_SYS_CHROOT",
+					"CAP_SYS_PTRACE",
+					"CAP_SYS_PACCT",
+					"CAP_SYS_ADMIN",
+					"CAP_SYS_BOOT",
+					"CAP_SYS_NICE",
+					"CAP_SYS_RESOURCE",
+					"CAP_SYS_TIME",
+					"CAP_SYS_TTY_CONFIG",
+					"CAP_MKNOD",
+					"CAP_LEASE",
+					"CAP_AUDIT_WRITE",
+					"CAP_AUDIT_CONTROL",
+					"CAP_SETFCAP",
+					"CAP_DAC_OVERRIDE",
+					"CAP_MAC_OVERRIDE",
+					"CAP_DAC_READ_SEARCH",
+					"CAP_MAC_ADMIN",
+					"CAP_SYSLOG",
+					"CAP_WAKE_ALARM",
+					"CAP_BLOCK_SUSPEND",
+					"CAP_AUDIT_READ"
+				]
+			},
+			"noNewPrivileges": false
+		},
+		"root": {
+			"path": "rootfs",
+			"readonly": true
+		},
+                "hooks": {},
+		"linux": {
+			"resources": {
+				"devices": [
+					{
+						"allow": true,
+						"access": "rwm"
+					}
+				]
+			},
+			"namespaces": [
+				{
+					"type": "mount"
+				}
+			]
+		},
+		"mounts": [
+		{
+			"destination": "/proc",
+			"type": "proc",
+			"source": "proc"
+		},
+		{
+			"type": "bind",
+			"source": "/dev",
+			"destination": "/dev",
+			"options": [
+				"rbind",
+				"rw",
+				"mode=755"
+			]
+		},
+		{
+			"type": "bind",
+			"source": "/dev/log",
+			"destination": "/dev/log",
+			"options": [
+				"rbind",
+				"rw",
+				"mode=755"
+			]
+		},
+		{
+			"destination": "/dev/pts",
+			"type": "devpts",
+			"source": "devpts",
+			"options": [
+				"nosuid",
+				"noexec",
+				"newinstance",
+				"ptmxmode=0666",
+				"mode=0620",
+				"gid=5"
+			]
+		},
+		{
+			"destination": "/dev/shm",
+			"type": "tmpfs",
+			"source": "shm",
+			"options": [
+				"nosuid",
+				"noexec",
+				"nodev",
+				"mode=1777",
+				"size=65536k"
+			]
+		},
+		{
+			"destination": "/sys",
+			"type": "bind",
+			"source": "/sys",
+			"options": [
+				"rbind",
+				"rw",
+				"mode=755"
+			]
+		},
+		{
+			"type": "bind",
+			"source": "/home",
+			"destination": "/home",
+			"options": [
+				"rbind",
+				"rw",
+				"mode=755"
+			]
+		},
+		{
+			"type": "bind",
+			"source": "/root",
+			"destination": "/root",
+			"options": [
+				"rbind",
+				"rw",
+				"mode=755"
+			]
+		},
+		{
+			"type": "bind",
+			"source": "/etc/firewalld",
+			"destination": "/etc/firewalld",
+			"options": [
+				"rbind",
+				"rw",
+				"mode=755"
+			]
+		},
+		{
+			"type": "bind",
+			"source": "/etc/sysconfig/firewalld",
```

Comment:
I saw this error in my log:

```
Failed to open temporary file: [Errno 30] Read-only file system: '/etc/sysconfig/network-scripts/ifcfg-ens3.xz8ngeo7'
```

I think we should bind mount the entire `/etc/sysconfig`